wifi-host-driver/WiFi_Host_Driver/src/bus_protocols/whd_bus_spi_protocol.c

```diff
@@ -1052,7 +1052,7 @@ static whd_result_t whd_spi_download_firmware(whd_driver_t whd_driver)
             ( (csr_val & SBSDIO_HT_AVAIL) == 0 ) &&
             (loop_count < ( uint32_t )HT_AVAIL_TIMEOUT_MS) )
     {
-        cy_rtos_delay_milliseconds( (uint32_t)1 );
+        cy_rtos_delay_milliseconds( (uint32_t) 2);
```

Comment:
Can we have a comment here on why the correct value is 2?

Reply:
@ThreeEights, can you reply here?

Reply:
@ifyall - I should have dug deeper to come up with a cleaner solution.  I measured the tiiming, and the CYW43439 on the Pico W took 1698 milliseconds for the HT wait loop to complete.  A more logical answer would be to increase HT_AVAIL_TIMEOUT_MS for that chip instead of mucking around inside the wait loop, so that's in the latest update.